v2.0/admin-ui-node-map.md

```diff
@@ -0,0 +1,185 @@
+
+To get the latitudes and longitudes of AWS regions, see [Locations Co-ordinates for Reference](#location-co-ordinates-for-reference).
+
+#### Step 5. Verify the settings 
+
+Navigate to `https://localhost:8080/#/reports/localities` and use the **Localities** table to verify that the latitudes and longitudes correspond to the correct regions.
+
+#### Step 6. View the Node Map
+
+[Navigate to the **Overview page**](admin-ui-cluster-overview.html) and switch to the **Node Map** view from the drop-down box.
+
+#### Step 7. Navigate the Node Map
+
+Suppose you want to navigate to Node 2, which is in datacenter `us-east-1` in the region `us-east`. To navigate to Node 2:
+
+1. Click on map component marked as **region=us-east** on the Node Map. The datacenter view is displayed.
+2. Click on the datacenter component marked as **datacenter=us-east-1**. The individual node components are displayed.
+3. To navigate back to the cluster view, either click on **Cluster** in the bread-crumb trail at the top of the Node Map, or click **Up to Cluster** in the lower left-hand side of the Node Map.
+
+## Understand the Node Map Components
+
+<img src="{{ 'images/admin-ui-node-map-components.png' | relative_url }}" alt="CockroachDB Admin UI Summary Panel" style="border:1px solid #eee;max-width:90%" />
+
+## Location Co-ordinates for Reference
+
+|  **Location** | **AWS Region** | **GCE Region Name** | **Azure Region** | **Latitude** | **Longitude** |
+|  ------ | ------ | ------ | ------ | ------ | ------ |
+|  US East (N. Virginia) | us-east-1 | us-east4 | East US | 37.478397 | -76.453077 |
+|  US East (Ohio) | us-east-2 |  |  | 40.417287 | -82.907123 |
+|  US Central (Iowa) |  | us-central1 | Central US |  |  |
+|  US West (N. California) | us-west-1 |  | West US | 38.837522 | -120.895824 |
+|  US West (Oregon) | us-west-2 | us-west1 |  | 43.804133 | -120.554201 |
+|  Canada (Central) | ca-central-1 | northamerica-northeast1 | Canada Central | 56.130366 | -106.346771 |
+|  EU (Frankfurt) | eu-central-1 | europe-west3 | Germany Central | 50.110922 | 8.682127 |
+|  EU (Ireland) | eu-west-1 |  | North Europe | 53.142367 | -7.692054 |
+|  EU (London) | eu-west-2 | europe-west2 | UK South | 51.507351 | -0.127758 |
+|  EU (Paris) | eu-west-3 |  |  | 48.856614 | 2.352222 |
+|  Asia Pacific (Tokyo) | ap-northeast-1 | asia-northeast1 | Japan East | 35.689487 | 139.691706 |
+|  Asia Pacific (Seoul) | ap-northeast-2 |  | Korea Central | 37.566535 | 126.977969 |
+|  Asia Pacific (Osaka-Local) | ap-northeast-3 |  | Japan West | 34.693738 | 135.502165 |
+|  Asia Pacific (Singapore) | ap-southeast-1 | asia-southeast1 | Southeast Asia | 1.352083 | 103.819836 |
+|  Asia Pacific (Sydney) | ap-southeast-2 | australia-southeast1 |  | -33.86882 | 151.209296 |
+|  Asia Pacific (Mumbai) | ap-south-1 | asia-south1 | West India | 19.075984 | 72.877656 |
+|  South America (São Paulo) | sa-east-1 | southamerica-east1 | Brazil South | -23.55052 | -46.633309 |
```

Comment:
maybe include these in SQL insert statement form, i.e.

```SQL
INSERT INTO system.locations VALUES
  ('region', 'us-east-1', 37.478397, -76.453077),
  ('region', 'us-east-2', 40.417287, -82.907123),
  ...
```

So people can just paste the entire table into the SQL shell if they want?

Reply:
@vilterp I tried doing it, but it made the table hideous. Now I am thinking of taking out the AWS, Azure, GCE codes and just having list of geographical regions and coordinates. Thoughts?